docs/earthpy-data-subsets.rst

```diff
@@ -0,0 +1,83 @@
+=====================
+EarthPy Data Subsets
+=====================
+
+EarthPy comes with the ability to download example datasets using
+``earthpy.data.get_data()``. These datasets are used in example documentation
+on how to use EarthPy, as well as in the EarthPy related lessons on
+earthdatascience.org. There are many keys that can be used to get data
+this way, all of which can be found below.
+
+Data Keys
+=========
+
+co-flood-extras
+---------------
+
+Contains two additional csv files for the Colorado Flood exercise on earthdatascience.org.
+Includes precipitation and `temperature <https://figshare.com/articles/Week_2_earth_analytics/4502138>`_
+data for the Boulder flood in 2013.
+
+colorado-flood
+--------------
+
+Main `dataset for the Colorado Flood <https://figshare.com/articles/Earth_Analytics_Python_Colorado_Flood_Teaching_Subset/6815945>`_
+exercise on https://www.earthdatascience.org. Contains precipitation,
+temperature, lidar, and clipping data for the 2013 Boulder Floods.
+
+spatial-vector-lidar
+--------------------
+
+Main `dataset for vector and lidar <https://figshare.com/articles/Week_4_Earth_Analytics_teaching_data_subset_-_lidar_and_insitu_for_California/4620268>`_
+lessons on https://www.earthdatascience.org. Contains shapefiles for
+a variety of exercises, including road, county, and country shapefiles. Also contains
+lidar data from NEON for research done in California, as well as supporting data for the lidar data.
+
+cold-springs-modis-h5
+---------------------
+
+`Modis satellite data <https://figshare.com/articles/Earth_Analytics_Cold_Springs_Fire_Remote_Sensing_Data/6083210>`_
+for the 2016 Cold Springs fire in h5 format.
+
+cold-springs-fire
+-----------------
+
+`Satellite and vector data for the 2016 Cold Springs fire <https://figshare.com/articles/Earth_Analytics_Cold_Springs_Fire_Remote_Sensing_Data/6083210>`_.
+Includes Landsat, Modis, and NAIP data.
+
+cs-test-naip
+------------
+
+Naip imagery related to the 2016 Cold Springs fire.
+
+cs-test-landsat
+---------------
+
+Landsat imagery related to the 2016 Cold Springs fire.
+
+ndvi-automation
+---------------
+
+`Landsat imagery and vector files <https://figshare.com/articles/Earth_Analytics_Python_HOMEWORK_Landsat_8_2017_for_SJER_and_HARV_Download_from_Earth_Explorer/7272500>`_
+for the automation assigment on https://www.earthdatascience.org. The data covers sites studied by NEON.
+
+vignette-landsat
+----------------
+
+Landsat imagery used in the vignette `gallery <https://earthpy.readthedocs.io/en/latest/gallery_vignettes/index.html>`_.
+
+vignette-elevation
+------------------
+
+Elevation data used in the vignette `gallery <https://earthpy.readthedocs.io/en/latest/gallery_vignettes/index.html>`_.
+
+california-rim-fire
+-------------------
+
+`Data related to 2013 California Rim fire <https://figshare.com/articles/Earth_Analytics_Applications_Data_for_Rim_Fire_California_2013/7749548>`_.
+Includes shapefiles and aerial imagery of the event.
```

Comment:
Again, lots of datasets described and listed on the Figshare description; bring that information here.

Reply:
For this dataset in particular, I mostly copy/pasted the description from the Figshare site. I'm not sure if that's plagarism and/or if that was what you were looking for. The description for this dataset was really good in particular. Let me know your thoughts!

Reply:
We definitely do not want to use others' words without crediting them. The descriptions on Figshare were written by either Leah or myself. Thanks for being for conscientious about this!